content/en/post/2024-04-12-changes-to-issuance-chains.md

```diff
@@ -0,0 +1,49 @@
+---
+author: Kruti Sutaria
+date: 2024-04-11T00:00:00Z
+slug: changes-to-issuance-chains
+title: "Deploying Let's Encrypt's New Issuance Chains"
+excerpt: "Using our new RSA & ECDSA intermediates to sign certificates starting June 6th."
+---
+
+On **Thursday, June 6th, 2024**, we will be switching issuance to use our [new intermediate certificates](https://letsencrypt.org/2024/03/19/new-intermediate-certificates#rotating-issuance). Simultaneously, we are removing the DST Root CA X3 cross-sign from our API, aligning with our strategy to shorten the Let's Encrypt chain of trust. We will begin issuing ECDSA end-entity certificates from a default chain that just contains a single ECDSA intermediate, removing a second intermediate and the option to issue an ECDSA end-entity certificate from an RSA intermediate. The Let's Encrypt staging environment will make an equivalent change on April 24th, 2024.
```

Comment:
I don't think the [new intermediate certificates] link should have the #rotating-issuance anchor

Reply:
Removed in 0ccfa5663a12f969467a80dc79544c8b3ed4529f

Reply:
Thank you!